<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="2">
    <widget class="QSpinBox" name="bottomOfRangeSpinBox">
     <property name="suffix">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="obscureBottomScreenPortionWidthLabel">
     <property name="text">
      <string>Hide Bottom of Image</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSlider" name="bottomOfRangeSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QRadioButton" name="fullStrokeRadioButton">
     <property name="text">
      <string>Full (up-and-down) stroke</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QRadioButton" name="halfStrokeRadioButton">
     <property name="text">
      <string>Half stroke</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="handyKeyEdit">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>Connection key</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="AnchorToBottomLabel">
     <property name="text">
      <string>Anchor to bottom</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="endingIntensitySpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1023</number>
     </property>
     <property name="value">
      <number>414</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="syncOffsetSlider">
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="endingIntensitySlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1023</number>
     </property>
     <property name="value">
      <number>414</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="endingIntensityLabel">
     <property name="text">
      <string>Ending Intensity</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="syncOffsetSpinBox">
     <property name="suffix">
      <string>ms</string>
     </property>
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSlider" name="handyScalingProportionSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="topOfRangeSpinBox">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="startingIntensityLabel">
     <property name="text">
      <string>Starting Intensity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="startingIntensitySlider">
     <property name="maximum">
      <number>1022</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="bottomOfRangeLabel">
     <property name="text">
      <string>Bottom of range</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="consistentLengthLabel">
     <property name="toolTip">
      <string>Try to keep all strokes the same length, increasing speed as needed</string>
     </property>
     <property name="text">
      <string>Consistent length</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="obscureBottomScreenPortionWidthSpinBox">
     <property name="suffix">
      <string>%</string>
     </property>
     <property name="minimum">
      <double>2.000000000000000</double>
     </property>
     <property name="maximum">
      <double>33.329999999999998</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>6.670000000000000</double>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="strokeTypeLabel">
     <property name="text">
      <string>For each beat:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="startingIntensitySpinBox">
     <property name="maximum">
      <number>1022</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="syncOffsetLabel">
     <property name="text">
      <string>Sync Offset</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="handyStrokeDurationSpinBox">
     <property name="suffix">
      <string>ms</string>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
     <property name="value">
      <number>420</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="connectToHandyCheckBox">
     <property name="text">
      <string>Connect to Handy</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSlider" name="topOfRangeSlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="obscureBottomScreenPortionWidthSlider">
     <property name="minimum">
      <number>200</number>
     </property>
     <property name="maximum">
      <number>3333</number>
     </property>
     <property name="value">
      <number>667</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLabel" name="AnchorToTopLabel">
     <property name="text">
      <string>Anchor to top</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QSlider" name="rangeAnchorSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="constantSpeedLabel">
     <property name="toolTip">
      <string>Move the stroker at a constant speed, shortening strokes as needed</string>
     </property>
     <property name="text">
      <string>Relative length</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="handyStrokeDurationLabel">
     <property name="text">
      <string>Duration of full up-and-down stroke</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="topOfRangeLabel">
     <property name="text">
      <string>Top of range</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="3">
    <widget class="QGroupBox" name="beatMeterOptionsGroup">
     <property name="title">
      <string>Beat Meter</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="1">
       <widget class="QSpinBox" name="meterMarkerDiameterSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="meterPixelsPerSecondLabel">
        <property name="text">
         <string>Speed (pixels per second)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="meterColouringModeTbdLabel">
        <property name="text">
         <string>Sorry - not done yet. Consider supporting me!</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="meterColouringModeLabel">
        <property name="text">
         <string>Colouring mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="meterHeightLabel">
        <property name="text">
         <string>Height (pixels)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="meterWidthLabel">
        <property name="text">
         <string>Width (pixels)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="meterSpeedSpinBox">
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="value">
         <number>167</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="meterWidthSpinBox">
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>1920</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="meterHeightSpinBox">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>50000</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="meterMarkerDiameterLabel">
        <property name="text">
         <string>Beat marker diameter (pixels)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="meterFrameRateLabel">
        <property name="text">
         <string>Frame rate (frames per second)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="meterFrameRateSpinBox">
        <property name="minimum">
         <double>10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>240.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.030000000000000</double>
        </property>
        <property name="value">
         <double>29.969999999999999</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
